docs/versionhistory.rst

```diff
@@ -6,6 +6,8 @@ APScheduler, see the :doc:`migration section <migration>`.
 
 **4.0.0a6**
 
+- Fixed an issue where `CronTrigger.next` returned a non-existing date on a DST change
+  (`#1059 <https://github.com/agronholm/apscheduler/issues/1059>`_)
```

Comment:
Add this under `**UNRELEASED**` as per the PR template instructions. Also code literals need two backticks on RST.
```suggestion
**UNRELEASED**

- Fixed an issue where ``CronTrigger.next()`` returned a non-existing date on a DST change
  (`#1059 <https://github.com/agronholm/apscheduler/issues/1059>`_)

**4.0.0a6**
```